src/config/AppConfig.ts

```diff
@@ -46,7 +46,7 @@ const defaultAppConfig: AppConfig = {
         MAX_SEED_ACCOUNTS_NUMBER: 10,
         MIN_PASSWORD_LENGTH: 8,
         SEED_ACCOUNT_NAME_PREFIX: 'SeedWallet-',
-        ANNOUNCE_TRANSACTION_TIMEOUT: 120000,
```

Comment:
TRANSACTION_TIMEOUT should be 120000

Reply:
Hi @bassemmagdy, i found that TRANSACTION_TIMEOUT is 240000 in integration branch[here](https://github.com/nemgrouplimited/symbol-desktop-wallet/blob/2eee5f6dc1d47eb64c893d522d5bca9f94da1b1d/src/config/AppConfig.ts#L49), should i change it to 120000 as your recommendation

Reply:
In your new pr, you shouldn't be facing such a kind of change. so ignore this for now.